planterbox_webdriver/screenshot.py

```diff
@@ -45,13 +45,20 @@ def capture_screenshot_delay(test, delay):
 
 @hook('before', 'feature')
 def set_save_directory(test):
-    """Sets the root save directory for saving screenshots.
+    """
+        Sets the root save directory for saving screenshots.
+    """
+
+    base = '{}/{}'.format(
+        test.config['screenshot.dir'][0],
+        test.config['start_time'][0],
+    )
```

Comment:
`os.path.join` can take multiple path segments to join together. Prefer that for joining path segments to `.format()`.